.github/workflows/csharp.yml

```diff
@@ -86,7 +88,7 @@ jobs:
   macos:
     name: AMD64 MacOS 10.15 C# ${{ matrix.dotnet }}
     runs-on: macos-latest
-    if: github.event_name == 'push'
+    if: !contains(github.event.pull_request.title, '[skip ci]')
```

Comment:
Do we need this change?

Reply:
We don't have many C# patches so this won't fill our queue.